### PR TITLE
package.json added

### PR DESCRIPTION
**Pull request by arshadkazmi42**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ak-url-validate",
```
Comment (bibekluitel) [inline package.json]: Name of package

Comment (arshadkazmi42) [reply package.json]: Updated

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "description": "[![Build Status](https://api.travis-ci.com/arshadkazmi42/ak-url-valdiate.svg?branch=master)](https://api.travis-ci.com/arshadkazmi42/ak-url-valdiate)",
```
Comment (bibekluitel) [inline package.json]: name of package

Comment (arshadkazmi42) [reply package.json]: I have removed description. We will add the description when we are getting this read for NPM release

**Pull request merged**